metrics/src/main/java/com/newrelic/telemetry/json/MetricBatchJsonTelemetryBlockWriter.java

```diff
@@ -0,0 +1,79 @@
+package com.newrelic.telemetry.json;
+
+import static java.lang.Double.isFinite;
+
+import com.newrelic.telemetry.Count;
+import com.newrelic.telemetry.Gauge;
+import com.newrelic.telemetry.Metric;
+import com.newrelic.telemetry.Summary;
+import com.newrelic.telemetry.Telemetry;
+import com.newrelic.telemetry.Telemetry.Type;
+import com.newrelic.telemetry.TelemetryBatch;
+import java.util.Collection;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class MetricBatchJsonTelemetryBlockWriter implements JsonTelemetryBlockWriter {
+
+  private final MetricToJson metricToJson;
+
+  public MetricBatchJsonTelemetryBlockWriter(MetricToJson metricToJson) {
+    this.metricToJson = metricToJson;
+  }
+
+  @Override
+  public <T extends Telemetry> void appendTelemetry(TelemetryBatch<T> batch,
```

Comment:
Maybe `appendTelemetryJson` to match the other interface method

Reply:
Good idea, fixed that.  Thanks!